gateway/1.4/modules/ROOT/pages/flex-conn-reg-run-up.adoc

```diff
@@ -21,11 +21,11 @@ include::partial$task-reg-run-flex-gateway.adoc[tags=prerequisites-heading;prere
 
 [[linux]]
 // Register and run with a username and password as a Linux service
-include::partial$task-reg-run-flex-gateway.adoc[tags=user-linux-heading;reg-command-intro;user-replace-content;environment-replace-content;replace-content;reg-command-heading;reg-command-1;linux-reg-command;user-reg-command;environment-reg-command;connected-reg-command;organization-reg-command;output-reg-command-linux;reg-command-2;after-reg;linux-after-reg;connected-after-reg;create-config-folder-file;config-content;linux-start-commands;gateway-connected]
+include::partial$task-reg-run-flex-gateway.adoc[tags=user-linux-heading;reg-command-intro;user-replace-content;environment-replace-content;replace-content;reg-command-heading;reg-command-1;linux-reg-command;user-reg-command;environment-reg-command;connected-reg-command;organization-reg-command;output-reg-command-linux;reg-command-2;after-reg;linux-after-reg;disconnected-after-reg;create-config-folder-file;config-content;linux-start-commands;gateway-connected]
 
 [[docker]]
 // Register and run with a username and password in a Docker container
-include::partial$task-reg-run-flex-gateway.adoc[tags=user-docker-heading;reg-command-intro;sub-coll-info;user-replace-content;environment-replace-content;replace-content;reg-command-heading;reg-command-1;docker-reg-command;user-reg-command;environment-reg-command;connected-reg-command;organization-reg-command;output-reg-command-docker;reg-command-2;after-reg;after-reg-2;connected-after-reg;start-command;gateway-connected]
```

Comment:
changed tag name because the content in tag `connected-after-reg` was about a disconnected state, not a connected state, which was a bit confusing